components/selection/README.md

```diff
@@ -150,21 +154,19 @@ The `d2l-selection-action` is an optional component that provides a button for a
 
 ## Selection Action Dropdown [d2l-selection-action-dropdown]
 
-The `d2l-selection-action-dropdown` is an optional component that provides a button opener for dropdown content associated with the selection component (ex. bulk actions). The `requires-selection` attribute may be specified to indicate that the opener should be non-interactive if nothing is selected.
+The `d2l-selection-action-dropdown` is an optional component that provides a button opener for dropdown content associated with the [selection control](#selectionmixin) (ex. bulk actions). The `requires-selection` attribute may be specified to indicate that the opener should be non-interactive if nothing is selected.
 
-<!-- docs: demo live name:d2l-selection-action-dropdown autoSize:false size:medium align:baseline -->
+<!-- docs: demo live name:d2l-selection-action-dropdown align:baseline -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/dropdown/dropdown-menu.js';
   import '@brightspace-ui/core/components/menu/menu.js';
-  import '@brightspace-ui/core/components/menu/menu-item.js';
   import '@brightspace-ui/core/components/selection/selection-action-dropdown.js';
 </script>
 <d2l-selection-action-dropdown text="Actions" requires-selection>
   <d2l-dropdown-menu>
     <d2l-menu label="Actions">
```

Comment:
The `d2l-selection-action-menu-item` demo does have a `requires-selection` example. Removing selection items from the `d2l-selection-dropdown` demo was a suggestion of Jeff's to make it more clear that this component is just the menu itself and you need the items below to actually add items to the list.

Reply:
Sorry I meant actually having `d2l-selection-input` items below that show that the `d2l-selection-action-dropdown` becomes disabled if `requires-selection` is applied and nothing is checked. I agree that this would clutter the demo a bit though as well.

Reply:
Oh okay now I understand. I will test it out to see how it looks and if it seems too cluttered.